### PR TITLE
feat(nats): added js and support arg without value

### PR DESCRIPTION
**Pull request by Naymi**

### CODE REVIEW

File: packages/modules/nats/src/nats-container.test.ts [hunk 1]
```diff
@@ -67,6 +67,43 @@ describe("NatsContainer", () => {
   });
   // }
 
+  // JetStream {
+  it("should start with JetStream ", async () => {
+    // enable JS
+    const container = await new NatsContainer().withJS().start();
```
Comment (cristianrgreco) [inline packages/modules/nats/src/nats-container.test.ts:73]: Is `JS` a well known acronym for Jetstream for Nats users?

**Comment by jonjomckay** [comment]
This would be helpful for us - we use JetStream, but we're unable to use `NatsContainer` at all at the moment, as we can't use `withArg` with a single parameter.

**Comment by cristianrgreco** [comment]
Hi @jonjomckay, I'm not familiar with NATS. If you were using the `NatsContainer` and came across a method `withJS`, would you understand that it is for enabling jetstream?

**Comment by jonjomckay** [comment]
@cristianrgreco `withJetStream` would be clearer to me. `JS` is probably a bit confusing, especially with this being a JavaScript library!

**Comment by cristianrgreco** [comment]
IDK if @Naymi is going to address the issues with the PR, but @jonjomckay if you want this feature then I'd happily accept a PR. A copy of this one is fine with `withJS` changed to `withJetStream`, with lint issues addressed.

**Comment by jonjomckay** [comment]
@cristianrgreco @Naymi I've opened up another PR with the linting and naming fixed: https://github.com/testcontainers/testcontainers-node/pull/877

**Pull request closed without merging**